test/coverage/BUILD

```diff
@@ -0,0 +1,115 @@
+# This file is generated by test/coverage/gen_build.sh. DO NOT EDIT. Rerun
```

Comment:
No that's fine. This plan sounds good (generate BUILD file during coverage run). +1